boards/arm/msp_exp432e401y_dk/msp_exp432e401y_dk_defconfig

```diff
@@ -0,0 +1,7 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2021 pipei.sandor@gmail.com
+
+CONFIG_BOARD_MSP_EXP432E401Y_DK=y
+CONFIG_SOC_SERIES_MSP432E4XX=y
+CONFIG_SOC_MSP432E401Y=y
+CONFIG_CORTEX_M_SYSTICK=y
```

Comment:
If MPU is supported, you should enable it here (CONFIG_ARM_MPU=y) and add also CONFIG_HW_STACK_PROTECTION=y

Reply:
Yes. I agree.